x/evm/keeper/migrate.go

```diff
@@ -49,6 +58,75 @@ func removeExternalTokenBurnerCode(ctx sdk.Context, ck chainKeeper) error {
 	return nil
 }
 
+func migrateCommandBatchSignature(ctx sdk.Context, ck chainKeeper, signer types.Signer, m types.MultisigKeeper) error {
+	commandBatchID := ck.getLatestSignedCommandBatchID(ctx)
+	for {
+		commandBatchMetadata := ck.getCommandBatchMetadata(ctx, commandBatchID)
+		if commandBatchMetadata.ID == nil {
+			break
+		}
+		commandBatchID = commandBatchMetadata.PrevBatchedCommandsID
+
+		sigID := hex.EncodeToString(commandBatchMetadata.ID)
+
+		if commandBatchMetadata.Status != types.BatchSigned {
+			continue
+		}
+
+		multisigInfo, found := signer.GetMultisigSignInfo(ctx, sigID)
+		signInfo, ok := multisigInfo.(*tsstypes.MultisigInfo)
+		if !found || !ok {
+			return fmt.Errorf("sign info %s not found", sigID)
+		}
+
+		key, ok := m.GetKey(ctx, commandBatchMetadata.KeyID)
```

Comment:
Not sure we want to return error when the key cannot be found since we are only migrating the active keys in the new module but nothing else. Also, need to add `app.mm.SetOrderMigrations` in `app.go`

Reply:
changed to break if key not found